themes/helm/layouts/partials/sidebar.html

```diff
@@ -25,19 +25,19 @@ <h1>
         </li>
 
         {{ range $docsSections }}
-        {{ range .Sections }}
-        {{ .Render "section-link" }}
+          <!-- {{ range .Sections }}
```

Comment:
Why is this part commented out and not just completely removed? We can get it back if needed from Git?